### PR TITLE
multiselect component additional requirements

### PR DESCRIPTION
**Pull request by injas-bigbinary**
@Tsudhishnair _a

### CODE REVIEW

**Comment by Tsudhishnair** [comment]
@injas-bigbinary 

- We need to specify a minimum height MultiSelect Input as `58px`.  Its specified in the [design System.](https://www.figma.com/file/GvxNW24y6lmKkrzdZCIz2e/Design-System?node-id=945%3A32546) 
Screenshot 2022-08-25 at 11 10 27 AM

- The default color of the label should be `grey600` . 

- Currently the create button is shown even if we are searching for a option that's available in the list. 
Screenshot 2022-08-25 at 11 19 50 AM

-  SearchText is not trimmed before searching. ( Seems like an issue with the SearchBar component in neeto-ui-rn )

- We can have some default message shown when the `showCreateOption` is false and when the search result is empty. 

- Currently when the user clicks on the red area the bottomsheet is not getting opened. That area should dynamically grow based on the tags shown right, instead of specifying a specific width. 
Screenshot 2022-08-25 at 11 53 04 AM

File: src/components/MultiSelect.js [hunk 1]
```diff
@@ -18,43 +33,71 @@ import {
   Typography,
 } from "@components";
 
+const Typography2 = styled.Text`
```
Comment (Tsudhishnair) [inline src/components/MultiSelect.js:36]: We can name this as `Typography` itself and remove the same from the above import and use it in the file.

File: src/components/MultiSelect.js [hunk 2]
```diff
@@ -226,17 +284,17 @@ export const MultiSelect = ({
...
-            label={item?.label || labelExtractor(item, index)}
-          />
-        </Container>
-      )
+      <Container py={12}>
+        <CheckBox
+          disabled={disabled}
+          checked={itemIndex !== -1}
+          onSelect={onPress}
+          // checkboxContainerProp={containerStyle}
```
Comment (Tsudhishnair) [inline src/components/MultiSelect.js:294]: We can remove this is not required.

**Review by Tsudhishnair — CHANGES_REQUESTED** [review]
@injas-bigbinary _a

**Comment by injas-bigbinary** [comment]
@Tsudhishnair _a

https://user-images.githubusercontent.com/87293196/186728953-d15f7f95-0214-46cd-a4b3-25acc8d85213.mov

**Comment by injas-bigbinary** [comment]
Screenshot_1661448197

**Comment by Tsudhishnair** [comment]
@injas-bigbinary I have fixed https://github.com/bigbinary/neeto-ui-rn/pull/448#discussion_r954562807 in the last commit.
Rest of the changes made LGTM. Merging.

**Pull request merged**